manager/integration/tests/test_recurring_job.py

```diff
@@ -476,13 +479,96 @@ def test_recurring_jobs_for_detached_volume():  # NOQA
     13. Verify that during the recurring backup, the volume's frontend is
         disabled, and pod cannot start.
     14. Wait for the recurring backup finishes.
-        Delete the recurring backup
-    15. In a 6-mintute retry loop, verify that the pod can eventually start.
-
-    16. Change the setting allow-recurring-job-while-volume-detached to false
-    17. cleanup
+        Delete the recurring backup.
+    15. In a 6-minutes retry loop, verify that the pod can eventually start.
+    16. Change the setting allow-recurring-job-while-volume-detached to false.
+    17. Cleanup.
     """
-    pass
+    recurring_job_setting = \
+        client.by_id_setting(SETTING_RECURRING_JOB_WHILE_VOLUME_DETACHED)
+    client.update(recurring_job_setting, value="true")
+
+    vol = common.create_and_check_volume(client, volume_name, size=str(1 * Gi))
+
+    lht_hostId = get_self_host_id()
+    vol.attach(hostId=lht_hostId)
+    vol = common.wait_for_volume_healthy(client, vol.name)
+
+    data = {
+        'pos': int(SIZE),
+        'content': common.generate_random_data(50 * Mi),
+    }
+    common.write_volume_data(vol, data)
+
+    vol.detach()
+    vol = common.wait_for_volume_detached(client, vol.name)
+
+    jobs = [
+        {
+            "name": RECURRING_JOB_NAME,
+            "cron": "*/1 * * * *",
+            "task": "backup",
+            "retain": 1
+        }
+    ]
+    vol.recurringUpdate(jobs=jobs)
+    common.wait_for_backup_completion(client, vol.name)
+    for _ in range(4):
+        bv = client.by_id_backupVolume(vol.name)
+        backups = bv.backupList().data
+        assert len(backups) == 1
+        time.sleep(60)
```

Comment:
I think 2 mins is enough, e.g. `time.sleep(30)`,  because the recurring backup job run every minute and we only need to make sure that it doesn't make a new backup in the next run. How do you think?

Reply:
Addressed.